detd/manager.py

```diff
@@ -221,6 +234,72 @@ def update_base_time(self, config):
         safety_margin = multiple * period
 
         config.stream.base_time = (now + ns_until_next_cycle) + safety_margin
+    
+    def add_listener(self, config):
+            '''
+            Performs the local configuration for the configuration provided
+            and returns the associated VLAN interface and socket priority
+            Parameters:
+                config: configuration
+            Returns:
+                VLAN interface
+                socket priority
+            '''
+
+            logger.info("Adding listener to InterfaceManager")
+
+
+            # Retrieve device rate
+            try:
+                rate = self.interface.rate
+            except RuntimeError:
+                logger.exception("Error while retrieving device rate")
+                raise
+
+
+            # Assign resources
+            soprio, tc, queue = self.mapping.assign_and_map(config.stream.pcp, self.scheduler.traffics)
+
+            traffic = Traffic(rate, TrafficType.SCHEDULED, config)
+            traffic.tc = tc
+
+
+            # Add stream to schedule
```

Comment:
On Rx side, we do not currently support the concept of a schedule. Hence, we can safely remove this section.